src/Visualization/Python/PlotTrajectories.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Plot XY trajectory from inspiral run, concatenating all segments
+
+import glob
+import os
+
+import click
+import h5py
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+@click.command()
+@click.argument("simulation_name", type=str)
+@click.option(
+    "-o",
+    "--output",
+    type=str,
+    help=(
+        'Output filename. Choose ".pdf" '
+        'or ".png". If not provided, the plot will be displayed in a '
+        "graphic window."
+    ),
+)
+@click.option(
+    "--force", is_flag=True, help="Force overwriting existing output file."
+)
+def plot_trajectory_command(simulation_name, output, force):
+    """Plot trajectories from inspiral simulation
+
+    Concatenates partial trajectories from each Segment and plots full
+    trajectories.
+    """
+    reductionfiles = select_reduction_files("./", simulation_name)
+    AhA, AhB = import_A_and_B(reductionfiles)
+    if AhA is not None and AhB is not None:
+        plot_trajectory(AhA, AhB, simulation_name, output, force)
+    else:
+        click.echo("Error: Unable to import data.")
+
+
+def select_reduction_files(dir_path, simname):
+    reduction_files_list = glob.glob(
+        os.path.join(dir_path, "**", "BbhReductions.h5"), recursive=True
+    )
+    selected_dir_list = [
+        path for path in reduction_files_list if "Segment" in path
+    ]
+    selected_file_list = [file for file in selected_dir_list if simname in file]
+    return selected_file_list
+
+
+def import_A_and_B(filenames):
+    A_data = []
+    B_data = []
+
+    for filename in filenames:
+        try:
+            with h5py.File(filename, "r") as file:
+                if "ApparentHorizons/ControlSystemAhA_Centers.dat" in file:
```

Comment:
Take the subfile names as options to the CLI command that have this as the default value.